Update documentation for AndroidX and Exoplayer 12 (#69)
Vualto/vudrm-public-docs: docs/source/integrations/mobile/android-widevine-sdk.md

## Known Issues

- Exoplayer 12 for AndroidX introduced a requirement that the PSSH box be present in the main manifest. If the PSSH box is not present in the main manifest then Exoplayers `DefaultDrmSessionManager` will throw this error `MissingSchemeDataException: Media does not support uuid: edef8ba9-79d6-4ace-a3c8-27dcd51d21ed`.

- 32-bit devices displayed issues where a license expiry time (secs) is too large to be handled, it therefore returns 0 seconds remaining and considers the license expired. To work around this, always set the license expiry time in your VUDRM token policy. For further information about VUDRM please contact us, or refer to our [documentation](https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html).

If you believe you have found any other issue, please contact us at support@vualto.com

## Troubleshooting

- Most issues are content related. You can use our demo application to test your own content by updating the `media.exolist.json` with your configurations.

- Errors may also arise because the stream or asset configuration is not correct. 

> 	- Tokens - You can easily eliminate token issues by beginning with a default Widevine token policy. Please ensure your token is formatted correctly and validates. For the avoidance of doubt, where tokens use dates, the dates should always be in the future. For further information about tokens please refer to our [documentation](https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html).

Reviewer: ```suggestion
	- Tokens - You can easily eliminate token issues by beginning with an empty policy in the VUDRM token. Please ensure your token is formatted correctly and validates. For the avoidance of doubt, where tokens use dates, the dates should always be in the future. For further information about tokens please refer to our [documentation](https://docs.vualto.com/projects/vudrm/en/latest/VUDRM-token.html).
```
